contracts/payloads/IGP29/PayloadIGP29.sol

```diff
@@ -0,0 +1,469 @@
+            "msg.sender-not-allowed"
+        );
+
+        uint256 totalActions = 1;
+        address[] memory targets = new address[](totalActions);
+        uint256[] memory values = new uint256[](totalActions);
+        string[] memory signatures = new string[](totalActions);
+        bytes[] memory calldatas = new bytes[](totalActions);
+
+        // Action 1: call executePayload on timelock contract to execute payload related to Fluid
+        targets[0] = address(TIMELOCK);
+        values[0] = 0;
+        signatures[0] = "executePayload(address,string,bytes)";
+        calldatas[0] = abi.encode(ADDRESS_THIS, "execute()", abi.encode());
+
+        uint256 proposedId = GOVERNOR.propose(
+            targets,
+            values,
+            signatures,
+            calldatas,
+            description
+        );
+
+        require(proposedId == PROPOSAL_ID, "PROPOSAL_IS_NOT_SAME");
+    }
+
+    function execute() external {
+        require(address(this) == address(TIMELOCK), "not-valid-caller");
+
+        /// Action 1: Approve fUSDC and fUSDT protocols to spend the reserves tokens
+        action1();
+
+       /// Action 2: Add config handler on liquidity layer for sUSDe/USDC & sUSDe/USDT vault.
+       action2();
+
+       /// Action 3: call cast() - transfer 100 stETH to Team Multisig from treasury.
+        action3();
+    }
+
+    function verifyProposal() external view {}
+
+    /***********************************|
+    |     Proposal Payload Actions      |
+    |__________________________________*/
+
+    /// @notice Action 1: Approve fUSDC and fUSDT protocols to spend the reserves tokens
+    function action1() internal {
+        address[] memory protocols = new address[](2);
+        address[] memory tokens = new address[](2);
+        uint256[] memory amounts = new uint256[](2);
+
+        // fUSDC
+        protocols[0] = F_USDC;
+        tokens[0] = USDC_ADDRESS;
+        amounts[0] = 290_000 * 1e6; // 290k USDC
+
+        // fUSDT
+        protocols[1] = F_USDT;
+        tokens[1] = USDT_ADDRESS;
+        amounts[1] = 290_000 * 1e6; // 290k USDT
+
+        FLUID_RESERVE.approve(protocols, tokens, amounts);
+    }
+
+    /// @notice Action 2: Add config handler on liquidity layer for sUSDe/USDC & sUSDe/USDT vault.
+    function action2() internal {
+        address VAULT_sUSDe_USDC_CONFIG_HANDLER = address(0xa7C805988f04f0e841504761E5aa8387600e430b);
+        address VAULT_sUSDe_USDT_CONFIG_HANDLER = address(0x7607968F40d7Ac4Ef39E809F29fADDe34C00A0A6);
+
+        AdminModuleStructs.AddressBool[]
+            memory configs_ = new AdminModuleStructs.AddressBool[](2);
+
+        configs_[0] = AdminModuleStructs.AddressBool({
+            addr: address(VAULT_sUSDe_USDC_CONFIG_HANDLER),
+            value: true
+        });
+
+        configs_[1] = AdminModuleStructs.AddressBool({
+            addr: address(VAULT_sUSDe_USDT_CONFIG_HANDLER),
+            value: true
+        });
+
+        LIQUIDITY.updateAuths(configs_);
```

Comment:
These must be set at VaultFactory as vault auth, see https://github.com/Instadapp/inst-governance/blob/main/contracts/payloads/IGP15/PayloadIGP15.sol#L661

Reply:
Okay!

Reply:
> Same as on [IGP15](https://github.com/Instadapp/inst-governance/blob/main/contracts/payloads/IGP15/PayloadIGP15.sol#L661), for the sUSDe v1.1.0 vaults we must set the ethena rate config handlers as vault auths at the Vault factory:
 Vault_sUSDe_USDC 0x3996464c0fCCa8183e13ea5E5e74375e2c8744Dd, config handler 0xa7C805988f04f0e841504761E5aa8387600e430b
 Vault_sUSDe_USDT 0xBc345229C1b52e4c30530C614BB487323BA38Da5, config handler 0x7607968F40d7Ac4Ef39E809F29fADDe34C00A0A6
These two handlers we will also have to add to config rebalance triggers from API if not present there yet (

Reply:
<img width="831" alt="image" src="https://github.com/Instadapp/inst-governance/assets/22830915/43474ff5-0955-4671-bac5-fe2a43ca17e9">